packages/enhanced/src/lib/sharing/ConsumeSharedPlugin.ts

```diff
@@ -124,6 +120,8 @@ class ConsumeSharedPlugin {
         packageName: item.packageName,
         singleton: !!item.singleton,
         eager: !!item.eager,
+        issuerLayer: item.issuerLayer ? item.issuerLayer : undefined,
+        requiredLayer: item.requiredLayer ? item.requiredLayer : undefined,
```

Comment:
Rename that to `layer` for consistency with the webpack `module.rules`, which have `issuerLayer` as matching condition and `layer` as action that sets the layer of a module.

Reply:
Agreed. Will do

Reply:
@sokra is there a way to "wrap" something in a issuer later?
For example, take next.js - when i `expose` a module, the transforms are based on issuerLayer, so how would one specify an exposed module from a specific layer if how next categorizes it is via issuerLayer, since there is no issuer in federation? 
Perhaps this is a separate discussion for another PR though